### ISSUE TITLE
Append "px" to numeric values for OBJECT css only

### ISSUE THREAD

**Issue by madou** [atlassian-labs/compiled-css-in-js#34]
Currently if you do this:

```
<div css={{ fontSize: 12 }} />
```

It'll resolve to:

```
font-size: 12 
```

It should resolve to:

```
font-size: 12px
```

Should work for all APIs.

Confirmed here the behaviour for emotion and styled: https://codesandbox.io/s/youthful-bardeen-qtjfb

**Comment by theKashey** [comment]
I might be not the great idea. What about `font-size:1` would mean `font-size:12px` -> `(X-y)*factor+px`. Ie `font-size:2` would be `14px`, or `16px`?

In the same way `margin:1` become `margin:4px` - always a multiple of X.
In the same way `color: blue` becomes `atlassian-blue`, and `grey` -> `N600`

**Comment by madou** [comment]
hey reasonable point - i think for now it just needs to behave the same as styled components + emotion to make migration easier.

next step could enable plugins and the like to enable the above ^

**Comment by theKashey** [comment]
Then you have to think how two `ccss` with different settings might coexists in one file.
Although I think my idea should be linked to the theme issue. But as long as the transformation of a "number" to a "unit" might play a bit role in theming - please __don't__ "fix" the current issue ASAP.

**Comment by madou** [comment]
fixed in 5a84023
